src/app/home-page/recent-events/recent-events.component.html

```diff
@@ -18,43 +18,37 @@
         <ng-template #defaultAvatar>
           <img [src]="gravatarService.gravatarUrlForMysteryPerson()" alt="User avatar" />
         </ng-template>
+        <ng-template #withUsername>
+          <strong>
+            <a class="no-underline" [routerLink]="'/users/' + event.initiatorUser?.username">
+              {{ event.initiatorUser?.username }}
+            </a>
+          </strong>
+        </ng-template>
+        <ng-template #withoutUsername>
+          <a class="no-underline"> Unknown user </a>
+        </ng-template>
         <div [ngSwitch]="event.type" class="my-3">
           <!-- Entries -->
           <div *ngSwitchCase="EventType.PUBLISHENTRY" class="truncate-text-2">
-            <strong
-              ><a class="no-underline" [routerLink]="'/users/' + event.initiatorUser?.username">{{
-                event.initiatorUser?.username
-              }}</a></strong
-            >
+            <ng-container *ngIf="event.initiatorUser?.username; then withUsername; else withoutUsername"></ng-container>
```

Comment:
No need to change because it was already like this, but this `ng-container` duplication can probably be removed by moving it outside of the `ngSwitchCase`s. Looks like there's only three cases where there's no initiator user mentioned. This might require some re-styling and re-arranging, so again, no need to change